tests/wpt/web-platform-tests/performance-timeline/po-observe.any.js

```diff
@@ -21,9 +25,12 @@
       obs.observe({entryTypes: ["this-cannot-match-an-entryType"]});
     });
     assert_throws(new TypeError(), function () {
-      obs.observe({entryTypes: ["marks","navigate", "resources"]});
+      obs.observe({entryTypes: ["marks","navigate","resources"]});
+    });
+    assert_throws(new TypeError(), function () {
+      obs.observe({others: true});
```

Comment:
This is testing what it is already tested [here](https://github.com/servo/servo/pull/18372/commits/5eabe5a03b2671802ed274b85d9d523ecd1b913c#diff-e5ad205a87105f2e36e178a96754aeecR9) (the absence of `entryTypes`). So I think it's ok if we don't add this new test.